### PR TITLE
Added recipe for bref/symfony-messenger-sns

### PR DESCRIPTION
**Pull request by Nyholm**
| Q             | A
| ------------- | ---
| License       | MIT

FYI @mnapoli

### CODE REVIEW

**Comment by Nyholm** [comment]
@symfony-flex-server review please

**Comment by Nyholm** [comment]
@symfony-flex-server review please

**Comment by Nyholm** [comment]
Thank you for the review. 

I want @sroze to be aware of this package. Could you also give a review on this recipe?

File: bref/symfony-messenger-sns/0.2/manifest.json [hunk 1]
```diff
@@ -0,0 +1,9 @@
+{
+    "copy-from-package": {
+        "config/": "%CONFIG_DIR%/packages/"
+    },
+    "env": {
+        "AWS_SNS_DSN": "enqueue://acme?topic[name]=my_sns_topic",
```
Comment (sroze) [inline bref/symfony-messenger-sns/0.2/manifest.json]: Where does `acme` come from? Can't we use a real default value?

Comment (Nyholm) [reply bref/symfony-messenger-sns/0.2/manifest.json]: "acme" is a reference to the php-enqueue config value. 

I could change it do `default` to match the recipe: https://github.com/symfony/recipes-contrib/blob/master/enqueue/enqueue-bundle/0.9/config/packages/enqueue.yaml

**Pull request closed without merging**